feature-group-definitions/array.yml

```
@@ -1,5 +1,5 @@
-name: Arrays
-description: Basic JavaScript arrays
+name: Arrays (initial support)
+description: Arrays are ordered lists of JavaScript values.
```

```suggestion
description: Ordered lists of JavaScript values.
```

Suggestion to avoid repeating the name of the feature in the description. The description should just _describe_ the feature.

---

I've made a breakdown of descriptions written so far in https://github.com/web-platform-dx/web-features/issues/532#issuecomment-1986073057 for discussion. I think that when the verb is "is" or "represent" that perhaps the active voice isn't doing any good, and perhaps we should have two styles, with some guide for which to prefer.

---

Let's close this without changes. I actually think my suggestion wasn't a good one and dropped a [comment](https://github.com/web-platform-dx/web-features/issues/532#issuecomment-1996892324) on 532 about why. Let's keep the long form.